src/adapter/src/coord/timestamp_selection.rs

```diff
@@ -203,6 +204,25 @@ impl TimestampProvider for Coordinator {
             .expect("id does not exist")
             .write_frontier
     }
+
+    fn acquire_read_holds(
+        &mut self,
+        id_bundle: &CollectionIdBundle,
+    ) -> ReadHolds<mz_repr::Timestamp> {
+        let read_holds = self
+            .acquire_read_holds(mz_repr::Timestamp::minimum(), id_bundle, false)
+            .expect("can acquire read holds");
```

Comment:
I think even that `precise = true` parameter is a stop-gap solution for something else. I believe Gabor added it to make some edge cases with `REFRESH EVERY` work.

Reply:
Yep, that's also my understanding. So perhaps the better solution would have been to introduce a new `acquire_precise_read_holds` method that wraps an infallible `acquire_read_holds`, instead of adding a parameter  that influences the expected return type.